pydantic/version.py

```diff
@@ -63,16 +63,16 @@ def version_info() -> str:
     return '\n'.join('{:>30} {}'.format(k + ':', str(v).replace('\n', ' ')) for k, v in info.items())
 
 
-def parse_mypy_version(version: str) -> tuple[int, ...]:
```

Comment:
Lol why did we do this before 🤦‍♀️

Reply:
Ahhh ok I see I guess it was actually sort of reasonable re new needed ignore comment